docs/integrations/sources/adroll.md

```diff
@@ -0,0 +1,45 @@
+# AdRoll
+AdRoll is an ecommerce marketing platform that gives growing D2C brands the power to connect with customers wherever they are.
+This connector allows you to extract data from various AdRoll APIs such as Advertisables , Ads , Campaigns , Strategies and many more
+Docs : https://developers.adroll.com/apis
+
+## Configuration
+
+| Input | Type | Description | Default Value |
+|-------|------|-------------|---------------|
+| `Authorization` | `string` | Person Access Token (Authorization).  |  |
+| `client_id` | `string` | Client ID. The key ID of your app |  |
+| `client_secret` | `string` | Client secret. The secret of your app |  |
+| `client_refresh_token` | `string` | Refresh token.  |  |
+| `oauth_access_token` | `string` | Access token. The current access token. This field might be overridden by the connector based on the token refresh endpoint response. |  |
+| `oauth_token_expiry_date` | `string` | Token expiry date. The date the current access token expires in. This field might be overridden by the connector based on the token refresh endpoint response. |  |
+
+## Streams
+| Stream Name | Primary Key | Pagination | Supports Full Sync | Supports Incremental |
+|-------------|-------------|------------|---------------------|----------------------|
+| Segments | segment_id | DefaultPaginator | ✅ |  ❌  |
+| Campaign | campaign_eid | DefaultPaginator | ✅ |  ❌  |
+| Strategy |  | No pagination | ✅ |  ❌  |
+| Strategies | eid | No pagination | ✅ |  ❌  |
```

Comment:
Strategies has a cursor pagination: https://developers.adroll.com/docs/adroll-campaign-api/1/routes/strategy/all/get

Reply:
I am not really sure what to enter in the `path` as next page cursor , as I mentioned in my previous comment. I will try to enter `end_date` and push it. If that doesnt work I'll need a bit of your help with that ...